modules/gitops-argo-cd-installation.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-argo-cd-installation_{context}"]
+= Installing Argo CD 
+
+[role="_abstract"]
```

Comment:
AFAIK this isn't actually required now

Reply:
@kalexand-rh can you confirm?

Reply:
+1, the `[role="_abstract"]` annotation isn't required. If you remove the annotation, the abstract looks compliant with the current guidelines.

Reply:
@DebarghoGhosh, I recommend [installing or updating the newdoc tool](https://github.com/redhat-documentation/newdoc/blob/main/README.md) to get the current mod doc templates. Let me know if you're using another source and I'll follow up to make sure it is current as well.